moc-nfs-democratic-csi/base/serviceaccounts/moc-nfs-democratic-csi-controller-sa/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- serviceaccount.yaml
```

Comment:
@larsks do you want to keep those nearly-empty kustomizations in here? There's no point for insisting on them, correct? :slightly_smiling_face:

Reply:
Fixed.